CONTRIBUTING.adoc

```diff
@@ -0,0 +1,184 @@
+** What was difficult for you?
+** Did you stumble on something because the steps weren’t clear?
+** Was a dependency not mentioned?
+
+*A few pointers and other considerations*
+
+* Think about how you can improve the documentation as a whole. For example, how can you fix the issue you found so that others don’t run into the same challenges?
+* Are there multiple places the issue could be fixed?
+** Are there other pages that you should apply your update?
+** Would it help if there was a link to more or related information on a page? On a related page?
+* Was the title or description misleading? Did you expect to find something different in that docs section?
+* If you find something and don’t have the bandwidth to open a PR, make us aware of the pain point and open an https://github.com/knative/docs/issues/new[Issue]
+
+=== Working group meetings
+
+The https://github.com/knative/community/blob/master/working-groups/WORKING-GROUPS.md#documentation[Knative Documentation Working Group] meets biweekly on Tuesdays at 9:30am PT. https://calendar.google.com/calendar/embed?src=knative.team_9q83bg07qs5b9rrslp5jor4l6s%40group.calendar.google.com[Click here] to see the exact dates on the Knative working group calendar.
+
+The Working Group meetings are used to discuss documentation strategy and policy; we expect individual PR review to happen mostly asynchronously via GitHub, and https://slack.knative.dev[Slack].
+
+=== Team interface
+
+Documentation is a collaboration between several different disciplines, including engineers writing code, technical writers providing clarity and structure, product managers and marketers providing vision and comparisons, and readers who may contribute ideas and requests for clarification or additional edits.
+
+With many different types of contributors, it’s useful to establish a ``team API'' or process for the contributors interacting. Note that this ``API contract'' is a living document, and records the current best practices for operating efficiently across timezones without requiring too much coordination.
+
+Generally, we expect the tech writing team to own much of the process and results of the documentation process, harnessing the energy of other contributors and using tech writers as expert resources to improve the quality of existing documentation rather than writing content from scratch.
+
+== Workflow
+
+There are roughly two workflows: writing / contributing new content, and refining and reviewing existing content.
+
+=== New Content
+
+We expect most new content to be written by the subject matter expert (SME) which would be the contributor who actually worked on the feature or example.
+
+When writing new content, keep the following in mind:
+
+* Write one <<type of documentation>> per page.
+* If possible, follow the style/template of other documents of the same type.
+* Focus mostly on technical correctness; structure and language should be roughly correct, but don't need heavy review.
```

Comment:
Might want to mention at this point that the content will be reviewed by tech writers?

Reply:
"...but don't need heavy review, tech writers in the Docs WG have you covered on that front."

Reply:
Given the rest of the context (refining existing content mentioned before and after this section), do we need to repeat that there will be a separate workflow that will clean up the structure and language?

Reply:
I guess not! defer to you :) I think it's good either way